tests/core/test_entities.py

```diff
@@ -184,52 +184,6 @@ def test_set_input_with_constructor():
     assert_near(person('salary', "2017-10"), [2000, 3000, 1600, 0, 0])
 
 
-def test_person_without_household():
```

Comment:
We already test the content of the error message [here](https://github.com/openfisca/openfisca-core/blob/7414070d634bef4040ec6e4ed246695f77bbfc16/tests/web_api/test_calculate.py#L50). What exactly would you like to test 🤔  ?